### ISSUE TITLE
Review: Excellent contribution

### ISSUE THREAD

**Issue by nicholst** [Brainhack-Proceedings-2015/Oconnor_HBM_INDICloud#1]
This project describes the copying of data from several open repositories to an Amazon S3 bucket.  While at first blush it may seem like a silly, trivial effort, many of the repositories offer their data in tar archives, not in a form that is ready to analyze or even view.  This project instead provides an excellent service for the community by making 4 different open data sources available in a common framework, and, for users of Amazon Web Services Cloud Computing Services, provides high-speed access to this data. 

The contribution is neatly typeset and free of typographical errors.  Only a few trivial suggestions
- The "GigaScience repository REFXXX." refeence should be set or removed
- The screenshot is so small as to strain the eyes; if space is not a concern, make it span 2 columns, or crop it so it just focuses on the upper left-hand corner.

Finally, I was hoping to see some reference/comment on the file structure of the respective projects.  Is it the case that the data for each repository are just the expanded tar files? Or has an attempt been made to harmonise the file structures of the different projects?  Of course [BIDS](http://bids.neuroimaging.io/) wasn't on the scene then, but an excellent project for 2016 would be converting all of these S3 versions of the repositories into BIDS format!

**Comment by nicholsn** [comment]
@DaveOC90 have you have a chance to incorporate these suggestions?

**Comment by DaveOC90** [comment]
Dear Reviewer,

Thank you for the kind words and suggestions. I have updated the manuscript. Figure 1 has been enlarged, and details about the organization of the data on the bucket have been added. With regard to the reference being incorrect, a GigaScience Repository reference has not yet been assigned but will be in the future. As such the authors wish to include it in the current manuscript as a bookmark. 

Regards,

Dave
